invokeai/frontend/web/public/locales/en.json

```diff
@@ -1689,6 +1689,34 @@
         "missingUpscaleModel": "Missing upscale model",
         "missingTileControlNetModel": "No valid tile ControlNet models installed"
     },
+    "stylePresets": {
+        "active": "Active",
+        "choosePromptTemplate": "Choose Prompt Template",
+        "clearTemplateSelection": "Clear Template Selection",
+        "createPromptTemplate": "Create Prompt Template",
+        "defaultTemplates": "Default Templates",
+        "deleteImage": "Delete Image",
+        "deleteTemplate": "Delete Template",
+        "deleteTemplate2": "Are you sure you want to delete this template? This cannot be undone.",
+        "editTemplate": "Edit Template",
+        "flatten": "Flatten selected template into current prompt",
+        "insertPlaceholder": "Insert placeholder",
+        "myTemplates": "My Templates",
+        "name": "Name",
+        "negativePrompt": "Negative Prompt",
+        "noMatchingTemplates": "No matching templates",
+        "placeholderDirections": "Use the button to specify where your manual prompt should be included in the template. If you do not provide one, the template will be appended to your prompt.",
```

Comment:
I think we can be a bit clearer in the message, maybe something like this. Can use a component to style the placeholder string

```suggestion
        "promptTemplatesDesc1": "Prompt templates add text to the prompts you write in the prompt box.",
        "promptTemplatesDesc2": "Use the placeholder string <Pre>{{placeholder}}</Pre> to specify where your prompt should be included in the template.",
        "promptTemplatesDesc3": "If you omit the placeholder, the template will be appended to the end of your prompt.",
```

In `StylePresetForm`:
```tsx
      <Box>
        <Text variant="subtext">{t('stylePresets.promptTemplatesDesc1')}</Text>
        <Text variant="subtext">
          <Trans
            i18nKey="stylePresets.promptTemplatesDesc2"
            components={{ Pre: <Pre /> }}
            values={{ placeholder: PRESET_PLACEHOLDER }}
          />
        </Text>
        <Text variant="subtext">{t('stylePresets.promptTemplatesDesc3')}</Text>
      </Box>
// Then create a simple component to handle formatting
const Pre = (props: PropsWithChildren) => (
  <Text as="span" fontFamily="monospace" fontWeight="semibold">
    {props.children}
  </Text>
);
```